pkg/util/common.go

```diff
@@ -534,3 +540,13 @@ func WrapText(originalText string, lineSize int) (wrappedText string) {
 
 	return wrappedText
 }
+
+// RandomString returns a random alphanumeric string.
+func RandomString(n int) string {
+	result := make([]byte, n)
+	for i := range result {
+		result[i] = CharSet[rnd.Intn(len(CharSet))]
+	}
+
+	return string(result)
+}
```

Comment:
Do we wanna use something like a standard UUID v4 instead of our own implementation?

Reply:
thanks for your review